### PR TITLE
mdelucasg-fromRDFtoJSON-LD.jsonld 

### PR DESCRIPTION
**Pull request by mdelucasg**
Antes había solicitado el PullREquest con un archivo equivocado.
Perdón.

### CODE REVIEW

File: Assignment2/mdelucasg-fromRDFtoJSON-LD.jsonld [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+"@context": {
+	"sample": "<http://example.org>",
+	"rdfs": "<http://www.w3.org/2000/01/rdf-schema#> ",
+	"properties": "http://example.org/Properties#>",
+	"xsd": "<http://www.w3.org/2001/XMLSchema#>"
+},
```
Comment (paoespinozarias) [inline Assignment2/mdelucasg-fromRDFtoJSON-LD.jsonld:7]: (This comment applies to all this file. Please, validate your code before submitting it. There are some syntax errors. You might use the json-ld playground online tool to test it. )

Your prefixes are wrong, check how they should be defined. Also, check the “sample” URI, something is missing at the end.

File: Assignment2/mdelucasg-fromRDFtoJSON-LD.jsonld [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+		[{"@id":"http://example.org/elementsUser10A"}]},
+	{"@id":"http://example.org/elementsMeasurement8401", "@type":"sample:elements","properties:hasTemperature":
+		[{"@value":"29","@type":"xsd:integer"}],
+		"http://example.org/elementsatTime":
+		[{"@value":"2010-06-12T12:00:12","@type":"xsd:dateTime"}]},
+	{"@id":"http://example.org/elementsSensor029", "@type":"sample:elements","properties:hasMeasurement":
+		[{"@id":"http://example.org/elementsMeasurement8401"}]},
+	{"@id":"http://example.org/elementsUser10A", "@type":"sample:elements","properties:hasName":
+		[{"@value":"Pedro"}]}
+]"
```
Comment (paoespinozarias) [inline Assignment2/mdelucasg-fromRDFtoJSON-LD.jsonld:21]: This way to define the JSON-LD is ok, but you are missing another tag that should be used when you want to define data inside a named graph (@graph).

Finally, remember that Class, Sensor, Computer, User, and Measurement are different objects (each of them has its own attributes) so provide different types for them (not use “sample:elements” for all).

File: Assignment2/mdelucasg-fromJSONtoJSON-LD.jsonld [hunk 3]
```diff
@@ -0,0 +1,33 @@
+{
```
Comment (paoespinozarias) [inline Assignment2/mdelucasg-fromJSONtoJSON-LD.jsonld:1]: Please, validate the code and fix the syntaxs errors.

**Review by paoespinozarias — CHANGES_REQUESTED** [review]
Please, validate your code before submmit it. You can use json-ld playground. There are syntax errors and also structure ones:

**Comment by mdelucasg** [comment]
I have checked all the syntax problems. Ignore the changes in the mdelucasg.ttl file (I don't know how to remove it from this pull request)

File: Assignment2/mdelucasg-fromJSONtoJSON-LD.jsonld [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+	"@context":"http://schema.org/",
+	"@type": "Person>",
```
Comment (paoespinozarias) [inline Assignment2/mdelucasg-fromJSONtoJSON-LD.jsonld:3]: Wrong definition, delete >

File: Assignment2/mdelucasg-fromJSONtoJSON-LD.jsonld [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+			"@type": "Place",
+   		"address": {
+  			"@type": "PostalAddress",
+			"addressLocality": "Yuncler",
+			"addressRegion": "Toledo",
+        "addressCountry": "Spain"
+		}
+	},
+	"alumniOf": {
+	 	"@type": "CollegeOrUniversity",
```
Comment (paoespinozarias) [inline Assignment2/mdelucasg-fromJSONtoJSON-LD.jsonld:15]: UPM is a University thus we can define it as "@type":"University"

File: Assignment2/mdelucasg-fromRDFtoJSON-LD.jsonld [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+	{"@id":"http://example.org/measureMeasurement8401", "@type":"sample:measure","properties:hasTemperature":
+		{"@value":"29","@type":"xsd:integer"},
+     	"properties:atTime":
+		{"@value":"2010-06-12T12:00:12","@type":"xsd:dateTime"}},
+ 
+	
+	{"@id":"http://example.org/userUser10A", "@type":"sample:user","properties:hasName":
+		{"@value":"Pedro"}}
+]
+}
```
Comment (paoespinozarias) [inline Assignment2/mdelucasg-fromRDFtoJSON-LD.jsonld:31]: It is ok, Well done!

**Review by paoespinozarias — CHANGES_REQUESTED** [review]

**Comment by mdelucasg** [comment]
Done it!

**Review by paoespinozarias — APPROVED** [review]

**Pull request merged**